README.md

```diff
@@ -4,6 +4,11 @@ API for serializing and de-serializing JavaScript objects using the
 addon compared to the native `JSON` object isn't too bad, and the space
 required for serialized data is far less than JSON.
 
+### Extension: support for undefined
+The standard msgpack protocol turns `undefined` into `null`.
+This branch adopts (creationix's extension)[https://github.com/creationix/msgpack-js-browser#extension]
+for `undefined` and uses for it the reserved value '0xc4'
```

Comment:
@godsflaw I'm using 0xc1 to identify undefined. Which per the spec is :
 (never used)	11000001	0xc1

So the only wrong thing is in `README.md` which should be updated.